### PR TITLE
Update es.po

### PR DESCRIPTION
**Pull request by agdg**
Update translations

### CODE REVIEW

File: remmina/po/es.po [hunk 1]
```diff
@@ -356,7 +356,7 @@ msgstr "Guardar %s"
 #: remmina-plugins/rdp/rdp_plugin.c:1122 remmina-plugins/nx/nx_plugin.c:692
 #: remmina-plugins/vnc/vnc_plugin.c:1995 remmina-plugins/vnc/vnc_plugin.c:2009
 msgid "User name"
-msgstr "Nombre de usuario"
+msgstr "Usuario"
```
Comment (Fitoschido) [inline remmina/po/es.po]: This change is completely wrong.

File: remmina/po/es.po [hunk 2]
```diff
@@ -374,7 +374,7 @@ msgstr "Guardar contraseña"
 
 #: remmina/src/remmina_init_dialog.c:465
 msgid "Certificate Details:"
-msgstr ""
+msgstr "Detalles del Certificado:"
```
Comment (Fitoschido) [inline remmina/po/es.po]: It is also a huge mistake when people does this Title Case thing in Spanish. Ugh.

File: remmina/po/es.po [hunk 3]
```diff
@@ -394,19 +394,19 @@ msgstr "¿Aceptar certificado?"
 
 #: remmina/src/remmina_init_dialog.c:561
 msgid "Certificate Changed! Details:"
-msgstr ""
+msgstr "El Certificado ha cambiado! Detalles:"
```
Comment (Fitoschido) [inline remmina/po/es.po]: Opening exclamation mark (¡) is missing.

File: remmina/po/es.po [hunk 4]
```diff
@@ -394,19 +394,19 @@ msgstr "¿Aceptar certificado?"
...
 
 #: remmina/src/remmina_init_dialog.c:606
 msgid "New Fingerprint:"
-msgstr ""
+msgstr "Nueva Huella Digital:"
 
 #: remmina/src/remmina_init_dialog.c:617
 msgid "Accept Changed Certificate?"
-msgstr ""
+msgstr "¿Aceptar los cambios del Certificado?"
```
Comment (Fitoschido) [inline remmina/po/es.po]: This is a sentence fragment (in English as well).

File: remmina/po/es.po [hunk 5]
```diff
@@ -849,7 +849,7 @@ msgstr "Desactivar icono en el área de notificación"
 
 #: remmina/src/remmina_pref_dialog.c:563
 msgid "Start Remmina in tray icon at user logon"
-msgstr ""
+msgstr "Mostrar el icono Remmina en el área de notificación al inicio"
```
Comment (Fitoschido) [inline remmina/po/es.po]: «icono Remmina» expresses a wrong idea, it is missing the particle _de_.

File: remmina/po/es.po [hunk 6]
```diff
@@ -1211,7 +1211,7 @@ msgstr "El cliente de escritorio remoto en GTK+"
 
 #: remmina/src/remmina_about.c:96
 msgid "translator-credits"
-msgstr ""
+msgstr "Créditos (Traductor)"
```
Comment (Fitoschido) [inline remmina/po/es.po]: This line is unnecessary.

**Comment by giox069** [comment]
@Fitoschido: can you provide a fixed es.po for the "next" branch ? I Agree with almost all of your comments. Please leave "Accept Changed Certificate?" as is: it's a big problem to fix it in English because it will modify all other languages. I also think that there are various Title Case problems in the newer es.po.
Also can you review "es.po" in the master branch, merged here in pull request #454 ?
Thank you.

**Pull request merged**